tests/test_lookml.py

```diff
@@ -1754,7 +1780,26 @@ def test_context_id(runner, glean_apps, tmp_path):
                             "sql": "${context_id}",
                         }
                     ],
-                }
+                },
+                {
+                    "dimensions": [
+                        {"name": "key", "sql": "${TABLE}.key", "type": "string"},
+                        {
+                            "hidden": "yes",
+                            "name": "position",
+                            "sql": "${TABLE}.position",
+                        },
+                        {"name": "value", "sql": "${TABLE}.value", "type": "number"},
+                    ],
+                    "name": "context__contexts",
+                },
+                {
+                    "dimensions": [
+                        {"name": "key", "sql": "${TABLE}.key", "type": "string"},
+                        {"name": "value", "sql": "${TABLE}.value", "type": "number"},
+                    ],
+                    "name": "context__contexts__position",
+                },
```

Comment:
Hmm, unless I'm missing something it looks like we're not actually testing explore generation in these tests? Might be good to add that (could be done as a followup).

Reply:
Right, those are just the views. I'll add a test here for checking the generated joins in the explore